scripts/jenkins/Makefile.jenkins

```diff
@@ -248,6 +248,9 @@ test-package-java:
 		h2o-jetty-9/build/libs/h2o-jetty-9-*.jar \
 		h2o-extensions/target-encoder/src/ \
 		h2o-extensions/target-encoder/build/libs/h2o-ext-target-encoder*.jar \
+		h2o-k8s/src/ \
+        h2o-k8s/build/libs/h2o-k8s-*.jar \
```

Comment:
knit: different indent than the rest of the list

Reply:
Fixed.